plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/glue/TrinoGlueCatalog.java

```diff
@@ -1056,4 +1153,29 @@ public Optional<CatalogSchemaTableName> redirectTable(ConnectorSession session,
         }
         return Optional.empty();
     }
+
+    com.amazonaws.services.glue.model.Table getTable(SchemaTableName tableName, boolean invalidateCaches)
```

Comment:
Is the default method privacy intentional?

Reply:
yes, this is indeed accessed from other classes in the package